Dockerfile

```diff
@@ -2,42 +2,74 @@
 #
 # VERSION               0.1
 
-FROM        ubuntu:15.04
-MAINTAINER  Matthias Vallentin <matthias@bro.org>
+FROM debian:buster-slim as builder
+LABEL maintainer="tobias.mayer@tenzir.com"
+LABEL builder=true
+
 
 ENV PREFIX /usr/local
-ENV CC clang
-ENV CXX clang++
+ENV CC clang-7
+ENV CXX clang++-7
+ENV BUILD_TYPE Release
+ENV BUILD_DIR /tmp/src
 
-# Compiler and dependcy setup
-RUN apt-get -qq update && apt-get -qqy install clang libc++-dev cmake git-core
+# Compiler and dependency setup
 RUN apt-get -qq update && apt-get -qqy install \
-    libpcap-dev libedit-dev libgoogle-perftools-dev
-RUN apt-get -qq update && apt-get -qqy install vim-tiny tmux wget
+    clang-7 libc++-dev libc++abi-dev cmake git-core \
+    libpcap-dev libssl-dev
 
 # By placing the ADD directive at this point, we build both CAF and VAST
 # every time. This ensures that the CI integration will always fetch a fresh
 # CAF tree, regardless of the Docker cache. The correct way to handle this
 # would be to provide a CAF docker image and use it in the FROM directive.
-ADD . $PREFIX/src/vast
+ADD . $BUILD_DIR/vast
 
 # CAF
-WORKDIR $PREFIX/src
+WORKDIR $BUILD_DIR
 RUN git clone https://github.com/actor-framework/actor-framework.git caf
 WORKDIR caf
-RUN git checkout develop
-RUN ./configure --prefix=$PREFIX --build-type=Release --no-examples --no-opencl
-RUN make
-RUN make test
-RUN make install
-RUN ldconfig
+RUN ./configure --prefix=$PREFIX --build-type=$BUILD_TYPE \
+    --no-examples --no-opencl --no-unit-tests --no-python && \
+    make -C build all install
+
+# Broker
+#WORKDIR $BUILD_DIR
+#RUN git clone --recurse-submodules https://github.com/zeek/broker.git
+#WORKDIR broker
+#RUN ./configure --prefix=$PREFIX --with-caf=$PREFIX --build-type=$BUILD_TYPE \
+#    --disable-python --disable-docs --disable-tests && \
+#    make -C build all install
```

Comment:
No Broker for now?

Reply:
I would prefer to follow up with this when it's needed.